apps/web/src/components/activity/ExecutionDetailsModal.tsx

```diff
@@ -56,7 +59,26 @@ export function ExecutionDetailsModal({
         }}
         data-test-id="execution-details-modal-loading-overlay"
       />
+
       <ExecutionDetailsAccordion steps={jobs} subscriberVariables={subscriberVariables} />
+      <When truthy={digestedNotificationId}>
+        <Center mt={20}>
+          <Text mr={10} size="md" color={colors.B60}>
+            Remaining execution has been merged to an active Digest.
+          </Text>
+        </Center>
+        <When truthy={onViewDigestExecution}>
+          <Center mt={10}>
+            <UnstyledButton
+              onClick={() => {
+                onViewDigestExecution && onViewDigestExecution(digestedNotificationId);
```

Comment:
```suggestion
                onViewDigestExecution(digestedNotificationId);
```
I think it is not necessary the check as this snippet won't render due the `When` component condition.

Reply:
I believe this was added because of typescript. Because that function is an optional field so possibly undefined. I guess our When component is not caught by it as a check

Reply:
That behaviour makes me to dislike that `When` component 😓

Reply:
Maybe we can somehow add it to the rules?